src/components/kubectl/logs.ts

```diff
@@ -86,7 +88,15 @@ function getLogsForContainer(
         return;
     }
 
-    kubectl.invokeInSharedTerminal(cmd);
+    const resource = `${podSummary.namespace}/${podSummary.name}`;
+    const panel = LogsPanel.createOrShow('TODO', 'Loading \u25CC', resource);
```

Comment:
Consider `...` rather than the 'dotted circle' character?

Reply:
done.